sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -15,10 +30,291 @@
+     */
+    public void defer(MessageLockToken lockToken, Map<String, Object> propertiesToModify) {
+        asyncClient.defer(lockToken, propertiesToModify).block(operationTimeout);
+    }
+
+    /**
+     * Moves a {@link ServiceBusReceivedMessage message} to the deadletter sub-queue.
+     *
+     * @param lockToken Lock token of the message.
+     *
+     * @throws NullPointerException if {@code lockToken} is null.
+     * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
+     *     mode.
+     * @throws IllegalArgumentException if {@link MessageLockToken#getLockToken()} returns a null lock token.
+     * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/service-bus-dead-letter-queues">Dead letter
+     *     queues</a>
+     */
+    public void deadLetter(MessageLockToken lockToken) {
+        asyncClient.deadLetter(lockToken).block(operationTimeout);
+    }
+
+    /**
+     * Moves a {@link ServiceBusReceivedMessage message} to the deadletter subqueue with deadletter reason, error
+     * description, and/or modified properties.
+     *
+     * @param lockToken Lock token of the message.
+     * @param deadLetterOptions The options to specify when moving message to the deadletter sub-queue.
+     *
+     * @throws NullPointerException if {@code lockToken} is null.
+     * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
+     *     mode.
+     * @throws IllegalArgumentException if {@link MessageLockToken#getLockToken()} returns a null lock token.
+     */
+    public void deadLetter(MessageLockToken lockToken, DeadLetterOptions deadLetterOptions) {
+        asyncClient.deadLetter(lockToken, deadLetterOptions).block(operationTimeout);
+    }
+
+    /**
+     * Reads the next active message without changing the state of the receiver or the message source. The first call to
+     * {@code peek()} fetches the first active message for this receiver. Each subsequent call fetches the subsequent
+     * message in the entity.
+     *
+     * @return A peeked {@link ServiceBusReceivedMessage}.
+     * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/message-browsing">Message browsing</a>
+     */
+    public ServiceBusReceivedMessage peek() {
+        return asyncClient.peek().block(operationTimeout);
+    }
+
+    /**
+     * Starting from the given sequence number, reads next the active message without changing the state of the receiver
+     * or the message source.
+     *
+     * @param fromSequenceNumber The sequence number from where to read the message.
+     *
+     * @return A peeked {@link ServiceBusReceivedMessage}.
+     * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/message-browsing">Message browsing</a>
+     */
+    public ServiceBusReceivedMessage peek(long fromSequenceNumber) {
+        return asyncClient.peek(fromSequenceNumber).block(operationTimeout);
```

Comment:
According to APIVIEW :  we should change to `peekAt` in sync and Async receiver.